x-pack/plugins/aiops/public/embeddables/change_point_chart/handle_explicit_input.tsx

```diff
@@ -8,23 +8,22 @@
 import type { CoreStart } from '@kbn/core/public';
 import { toMountPoint } from '@kbn/react-kibana-mount';
 import React from 'react';
-import type { EmbeddableChangePointChartExplicitInput } from './types';
-import type { AiopsAppDependencies } from '..';
-import { AiopsAppContext } from '../hooks/use_aiops_app_context';
-import type { AiopsPluginStartDeps } from '../types';
+import type { ChangePointEmbeddableState } from './types';
+import type { AiopsAppDependencies } from '../..';
+import { AiopsAppContext } from '../../hooks/use_aiops_app_context';
+import type { AiopsPluginStartDeps } from '../../types';
 import { ChangePointChartInitializer } from './change_point_chart_initializer';
-import type { EmbeddableChangePointChartInput } from './embeddable_change_point_chart';
 
 export async function resolveEmbeddableChangePointUserInput(
   coreStart: CoreStart,
   pluginStart: AiopsPluginStartDeps,
-  input?: EmbeddableChangePointChartInput
-): Promise<EmbeddableChangePointChartExplicitInput> {
+  input?: ChangePointEmbeddableState
```

Comment:
Would it be possible to rename this file? `ExplicitInput` is a concept that is very much tied to the old system.

Reply:
renamed in https://github.com/elastic/kibana/commit/9323cbf5b38253e1344e7729f9d18b3c09ade952